### PR TITLE
Add MIB for Netonix switches; widely used by ISPs

### PR DESCRIPTION
**Pull request by avantops**
NB: When building it appears that EXTRAS/reports/all has many
un-checked-in changes for Cisco and Arista.  Followed the procedure
at wiki/Updating-MIBs

### CODE REVIEW

File: EXTRAS/reports/all [hunk 1]
```diff
@@ -506766,6 +506793,7 @@
                   snmpSetGroup(5) type=22
                   systemGroup(6) type=22
                   snmpBasicNotificationsGroup(7) type=23
+                  netonixSwitchGroup(8) type=22
```
Comment (inphobia) [inline EXTRAS/reports/all]: only issue imo, seems to overwrite snmpGroup on the next line?

**Comment by inphobia** [comment]
seems to be ok except for 1 line (EXTRAS/reports/all line 506796 in your diff).

seems this comes from the .dat file line 18, not sure of this is wanted/required or not.

(registed on netonix.com & downloaded latest mib to check)

**Comment by avantops** [comment]
As noted above when we built from master `EXTRAS/reports/all` had many un-checked-in changes for Cisco and Arista; we hand edited `EXTRAS/reports/all` in an attempt to restrict it to just Netonix related changes, but we may have made mistakes as we are not really familiar with the format of the file.  Would you like us to update the branch to remove the one line noted?

**Comment by inphobia** [comment]
since it was a small mib i gave it a go myself. i can't say what the issue is with the cisco & arista changes you're seeing, but i can say the netorix mibs were prepared and submitted correcty by you. i'm seeing the same issue when i just tried to add the mib, it seems netorix wants to overwrite snmpMIBGroups.8 from SNMPv2-MIB.

i don't think that will have a big impact but this is not great design.

while i'm no expert i don't really see a reason to why they would want to use that specific oid. as such i'm inclined to put this on hold for a few days until someone with some more experience can verify.

**Comment by inphobia** [comment]
tagging @JeroenvIS , since i'm not sure what the best way to continue forward with this. 

what would be the preferred approach for:
> i'm seeing the same issue when i just tried to add the mib, it seems netorix wants to overwrite snmpMIBGroups.8 from SNMPv2-MIB.

**Comment by ollyg** [comment]
We can patch the MIB file to use `snmpMIBGroups.8` and remove the override

**Pull request merged**